ast/version_index.json

```diff
@@ -1367,6 +1367,13 @@
     }
   },
   "features": {
+    "rego_v1_import": {
+      "Major": 0,
+      "Minor": 59,
+      "Patch": 0,
+      "PreRelease": "",
+      "Metadata": ""
```

Comment:
What's metadata here? Seem like it would be good to have a description of what this means, but perhaps this is used for something else?

Reply:
Don't know, to be honest. This is generated by https://github.com/open-policy-agent/opa/blob/16fccc9a76f582156f5f5a3df85eb680dcdf9353/internal/cmd/genversionindex/main.go#L60, which is out of scope for this PR.